pyrokinetics/local_geometry.py

```diff
@@ -38,3 +38,19 @@ def load_from_gk_file(self, pyro, gk_code=None):
         """
 
         pass
+
+    def __deepcopy__(self, memodict={}):
```

Comment:
Avoid mutable default arguments

```suggestion
    def __deepcopy__(self, memodict):
```